httpcore5/src/main/java/org/apache/hc/core5/http/impl/nio/LengthDelimitedDecoder.java

```diff
@@ -86,7 +86,7 @@ public int read(final ByteBuffer dst) throws IOException {
             setCompleted();
             if (this.len < this.contentLength) {
                 throw new ConnectionClosedException(
-                                "Premature end of Content-Length delimited message body (expected: %,d; received: %,d)",
+                                "Premature end of Content-Length delimited message body (expected: %d; received: %d)",
```

Comment:
I do not understand why you'd want to make the message less readable by humans.

Reply:
As discussed, it is not consistent across locales. Thus the messages are never stable.